templates/al2/provisioners/install-worker.sh

```diff
@@ -196,7 +196,7 @@ EOF
 ###############################################################################
 
 sudo yum install -y nerdctl
-sudo mkdir /etc/nerdctl
+sudo -p mkdir /etc/nerdctl
```

Comment:
the `-p` flag was misplaced :d
```suggestion
sudo mkdir -p /etc/nerdctl
```